README.adoc

```diff
@@ -222,20 +185,22 @@ Successfully built ee99661593eb
 Successfully tagged ol-runtime:latest
 ----
 
-Each step of the build has a unique ID, which is an ID of an intermediate image. For example, step 2
+Each step of the build has a unique ID, which represents the ID of an intermediate image. For example, step 2
 has the ID `dfef8073fe42`, and step 4 has the ID `ee99661593eb`, which is also the ID of the final image.
 During the first build of your image, Docker caches every new layer as a separate image and reuses them
-for future builds for steps that didn't change. For example, if you run the build again, Docker reuses
-the images that it cached for steps 2 - 4. However, you can tweak the `CMD` instruction and change the
-`defaultServer` server to the `testServer` server, and then Docker reuses the intermediate images for only steps 2 - 3.
+for future builds for layers that didn't change. For example, if you run the build again, Docker reuses
+the images that it cached for steps 2 - 4. However, if you make a change in your Dockerfile, Docker would
+need to rebuild the subsequent layer since it also changed.
 
-You can pass in the `--no-cache=true` command into the build to disable the caching of intermediate images:
+You can however completely disable the caching of intermediate layers by running the build with the
+`--no-cache=true` flag:
```

Comment:
"However, you can..."

Reply:
I added "However, you can also...", but good otherwise